hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -90,6 +92,8 @@
   private static final String HIVE_LOCK_CHECK_MAX_WAIT_MS = "iceberg.hive.lock-check-max-wait-ms";
   private static final String HIVE_ICEBERG_METADATA_REFRESH_MAX_RETRIES = "iceberg.hive.metadata-refresh-max-retries";
   private static final String HIVE_TABLE_LEVEL_LOCK_EVICT_MS = "iceberg.hive.table-level-lock-evict-ms";
+  private static final String HIVE_TABLE_PARAMETER_SIZE_MAX = "iceberg.hive.table.parameter.size.max";
+  private static final long HIVE_TABLE_PARAMETER_SIZE_MAX_DEFAULT = 32672;
```

Comment:
Should we add a comment, giving some reference to when it was changed in Hive?

Reply:
Thanks, nit: how about "iceberg.hive.max.table.parameter.size".

Also, regarding the comment, it's kind of based on backend but Hive had a global limit of 4000 regardless of database to allow it to support across different database, so maybe "For Hive versions below 2.3, table parameter values should be capped at 4000 characters, see https://issues.apache.org/jira/browse/HIVE-12274"?

Reply:
Made the change

Reply:
one more nit: can the variable match the changed string:  HIVE_MAX_TABLE_PARAMETER_SIZE

Reply:
Actually sorry about this , I just noticed other flags, weird how they do dash after "iceberg.hive".  How about 'iceberg.hive.table-property-max-size"

Reply:
Made the change.